src/main/java/net/mattlabs/skipnight/Vote.java

```diff
@@ -303,7 +309,13 @@ public void addNo(UUID uuid, VoteType voteType) {
             if (voters.contains(voter)) {
                 voter = voters.get(voters.lastIndexOf(voter));
                 if (voter.getVote() == 0) {
-                    if (this.voteType == VoteType.NIGHT && Bukkit.getPlayer(uuid).getStatistic(Statistic.TIME_SINCE_REST) > 72000) {
+                    boolean playerMustSleep;
+                    // Check version for TIME_SINCE_REST added in 1.13
+                    if (Versions.versionCompare("1.13.0", version) <= 0) {
+                        playerMustSleep = player.getStatistic(Statistic.TIME_SINCE_REST) >= 72000;
```

Comment:
This is checking the statistic for the player who started the vote, not the player who just voted. It should be:
```java
Bukkit.getPlayer(uuid).getStatistic(Statistic.TIME_SINCE_REST) >= 72000;
```